microsite/static/css/custom.css

```
@@ -48,6 +48,14 @@ h6 {
   color: $textColor;
 }
 
+h2:hover .hash-link{
```

As I noted in #3707 , once you run `yarn prettier --write *`, this file will be re-written, an dthen you need to then re-commit, and push that change up into this AdnDozez:master branch.

Alternatively, the easy fix may just be this...

```suggestion
h2:hover .hash-link {
```